### PR TITLE
Add line separator on side bar for #631

### PR DESCRIPTION
**Pull request by mic-smith**

### CODE REVIEW

File: src/assets/styles/custom-scroll.less [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 .side-scroll {
   padding-bottom: 0px;
+  border-top: 1px solid @dark-background;
+  transition: border 0.7s;
```
Comment (WPFerg) [inline src/assets/styles/custom-scroll.less]: Could this be replaced by the `.transition` mixin?

Comment (owennw) [inline src/assets/styles/custom-scroll.less]: There's also a specific bit of LESS in search.less for something similar:

  ```.sidetab > div:first-of-type search .search-result {
    border-top: @search-splitter-style;
  }```

Can something be refactored to make these more common?

Comment (mic-smith) [inline src/assets/styles/custom-scroll.less]: @WPFerg Could replace with the transition mixin - though not sure what it adds and this is more explicit that it's only the border being animated?

@owennw That selector doesn't match anything. I'll remove

Comment (owennw) [inline src/assets/styles/custom-scroll.less]: @mic-smith Doesn't that selector create the top border that we do have on the search tab?

Comment (WPFerg) [inline src/assets/styles/custom-scroll.less]: @mic-smith It doesn't really add much, it just adds a bit of consistency -- but it's certainly not something I'd insist on

**Comment by mic-smith** [comment]
Rebased on dev and repushed

**Comment by owennw** [comment]
LGTM.

**Pull request merged**